src/Dependencies/CodeAnalysis.Debugging/Microsoft.CodeAnalysis.Debugging.Package.csproj

```diff
@@ -2,7 +2,7 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
```

Comment:
>netstandard1.3 [](http://example.com/codeflow?start=22&length=14)

These projects produce source package. Bumping the TFM means customers using the package won't be able to target netstandard1.3 anymore. Is that intentional?

Reply:
Hmm ... it wasn't intentional. At the same time is there any remaining realistic reason to support this? 

Unsure of the exact semantics of these packages. But given that everything is based on `netstandard2.0` now want to make sure we think supporting `netstandard1.3` here is still valuable.

Reply:
The policy for packages in general is to not update unless we need/want to use features of the newer platform. I am not against requiring `netstandard2.0` if it unlocks APIs that would be beneficial to use in the source.

Reply:
Well we wouldn't be in this mess that required this PR if SCI / SRM had followed that policy 😄

Reply:
Moved back to `netstandard1.3` and it didn't impact any of our build scenarios that I can see so reverted this part of the change.